src/karmen_frontend/src/actions/printers/reading.js

```diff
@@ -214,6 +214,12 @@ export const getWebcamSnapshot = createHttpAction(
   (orguuid, uuid, { dispatch, getState }) => {
     return denyWithNoOrganizationAccess(orguuid, getState, () => {
       let { printers } = getState();
+
+      // This can happen during organization switching and printer's detail opened. Just catch it silently.
```

Comment:
This is not true. This error happens because some bad designed component is trying to read organization after the user logged out. This will hide the error but will not fix it.

The source of the error seems in [reading.js:86 -> setTimeout](https://github.com/fragaria/karmen/blob/master/src/karmen_frontend/src/actions/printers/reading.js#L86) which is never cleared after logout.